clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/security/DcosAccountCredentials.groovy

```diff
@@ -0,0 +1,160 @@
+package com.netflix.spinnaker.clouddriver.dcos.security
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.netflix.spinnaker.clouddriver.dcos.cache.Keys
+import com.netflix.spinnaker.clouddriver.dcos.deploy.util.id.MarathonPathId
+import com.netflix.spinnaker.clouddriver.security.AccountCredentials
+
+import static com.netflix.spinnaker.clouddriver.dcos.DcosConfigurationProperties.LinkedDockerRegistryConfiguration
+
+class DcosAccountCredentials implements AccountCredentials<DcosCredentialMap> {
+  private static final String CLOUD_PROVIDER = Keys.PROVIDER
+
+  // TODO This clobbers the Instance#name after a change in gate to merge this credential information directly into
```

Comment:
Is this still an issue?  I thought that was fixed in https://github.com/spinnaker/gate/pull/364